src/components/Footer/Footer.vue

```diff
@@ -1,27 +1,28 @@
 <template>
 	<footer class="bg-primary-900">
-		<div class="container mx-auto text-gray-500 font-hairline flex items-center p-10">
-			<ul class="w-1/3">
+		<div class="container flex flex-col items-center sm:items-stretch sm:flex-row justify-between mx-auto text-gray-500 
```

Comment:
Please add more margin from the top and bottom.

![image](https://user-images.githubusercontent.com/27869979/109662197-1640e180-7b6b-11eb-9d22-34d3f77f59b4.png)